File: frontend/src/types/address.ts
// types/address.ts
export interface AddressVO {
  id: number;
  userId: number;
  receiverName: string;     // 收货人
  phone: string;         // 手机号
  province: string;      // 省
  city: string;          // 市
  district: string;      // 区
  detailAddress: string;        // 详细地址
  isDefault: boolean;    // 是否默认地址
  createTime?: Date;
  updateTime?: Date;
}

File: backend/src/main/java/com/demo/light/bean/Address.java
package com.demo.light.bean;

import lombok.*;

import java.time.LocalDateTime;
@Setter
@Getter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class Address {
    private Integer id;
    private Long userId;
    private String receiverName;
    private String phone;
    private String province;
    private String city;
    private String district;
    private String detailAddress;
    private Boolean isDefault;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}
